### DISCUSSION TITLE
Problem in following UMEP tutorial

### DISCUSSION THREAD

**Discussion by croostanse** [UMEP-dev/UMEP#517] [Q&A] [answered]
I seem to have found a problem in following the tutorial [A First QGIS and UMEP activity](https://umep-docs.readthedocs.io/projects/tutorial/en/latest/Tutorials/DailyShading.html)

When I come to the Calculate daily shadow patterns I got this message when I tried to follow the instructions but I only got this message and tried to follow it but could't get it working. 
![problem_umep](https://user-images.githubusercontent.com/132560042/236192157-3ccc73d8-37e6-4aa4-bddd-5cf81f4504d9.PNG)

With the help of a colleague I got it working. I needed to go through processing >Toolbox >UMEP > Processor > solar Radiation instead of UMEP -> Processor -> Solar radiation -> Daily Shadow Pattern. 

the window for Solar Radiation: Shadow Generator seems to have change too. 
This is what it looks today: 
![shadow generator](https://user-images.githubusercontent.com/132560042/236193927-1edb1fff-d142-49d0-a099-af1f2f15e53d.PNG)

This is what it looks like in the tutorial: 
![shadow generator tutorial](https://user-images.githubusercontent.com/132560042/236194489-def6ba3f-f807-4df8-b983-5a2949327861.PNG)

I dont know if this is the right place to post this. But is it possible to get the tutorial updated?

**Comment by biglimp** [comment, marked as answer]
Thanks. We will update the tutorial when we find the time.

The shadowing tool works just the same even though it moved to Processing. Should be able to do the tutorial anyway.

**Reply by croostanse** [reply]
No problem. Thank you.
